lib/actions/addLinkToProjectAction.js

```diff
@@ -0,0 +1,28 @@
+class AddLinkToProjectAction {
+
+  // validate(){}
+
+  async execute([label, link], project){ 
+    // project.links[label] = link;
+    try {
+      //store data -- > await db.store() if it is async
+      return new AddLinkToProjectResponse(label, link, project)
+    } catch(err) {
+        console.log(err);
```

Comment:
Catching and console logging means you need to "notice" the output in the console. Let these throw. Better to crash and know there's a problem than hide that problem.